crates/transcode/Cargo.toml

```diff
@@ -28,16 +30,18 @@ itertools = "0.10.5"
 tracing = "0.1.37"
 nom = "7.1.3"
 nom-supreme = { version = "0.7.0", features = ["error"] }
+primitive-types = "0.12.1"
 scale = { package = "parity-scale-codec", version = "3.0.0", features = ["derive"] }
 scale-info = { version = "2.3.1", default-features = false, features = ["derive"] }
 serde = { version = "1.0.154", default-features = false, features = ["derive"] }
 serde_json = "1.0.94"
-sp-core = "16.0.0"
-sp-runtime = "18.0.0"
+thiserror = "1.0.39"
 
 [dev-dependencies]
 assert_matches = "1.5.0"
 ink = "4.0.1"
+sp-core = { version = "18.0.0", default-features = false }
```

Comment:
Is `default-features = false` needed here?

Reply:
No, probably not.

Reply:
Tacked it on to https://github.com/paritytech/cargo-contract/pull/1026